crypto/evp/m_sigver.c

```diff
@@ -135,6 +135,14 @@ int EVP_DigestSignFinal(EVP_MD_CTX *ctx, unsigned char *sigret,
     return 1;
 }
 
+int EVP_DigestSign(EVP_MD_CTX *ctx, unsigned char *sigret, size_t *siglen,
+                   const unsigned char *tbs, size_t tbslen)
+{
+    if (sigret != NULL && EVP_DigestSignUpdate(ctx, tbs, tbslen) <= 0)
```

Comment:
This can be legitimate concern, because EVP_DigestSignUpdate simply ctx->update and there is no guarantee that underlying subroutine checks for tbs being NULL. However, not checking is kind of consistent with current state of affairs in sense that this part of EVP kind of assumes that caller performed the check...